cmd/ndm_daemonset/filter/osdiskexcludefilter_test.go

```diff
@@ -70,16 +71,25 @@ func TestOsDiskExcludeFilterExclude(t *testing.T) {
 		expected bool
 	}{
 		"exclude path is /dev/sda and device path is /dev/sda": {
-			filter: oSDiskExcludeFilter{excludeDevPath: "/dev/sda"},
+			filter: oSDiskExcludeFilter{excludeDevPath: []string{"/dev/sda"}},
 			disk: &blockdevice.BlockDevice{
 				Identifier: blockdevice.Identifier{
 					DevPath: "/dev/sda",
 				},
 			},
 			expected: false,
 		},
-		"exclude path is /dev/sda and  device path is /dev/sda1": {
-			filter: oSDiskExcludeFilter{excludeDevPath: "/dev/sda"},
+		"exclude path is /dev/sda and device path is /dev/sda1": {
```

Comment:
Will it be possible to add new test cases, instead of modifying existing cases?

Reply:
@akhilerm seems like this is a new test case I added, can you hceck once more?

Reply:
Older one's have to be modified to have slice rather than string